slushfile.js

```diff
@@ -275,8 +292,8 @@ gulp.task('configGulp', ['init'], function(done) {
     configJSON['ml-host'] = settings.marklogicHost;
     configJSON['ml-admin-user'] = settings.marklogicAdminUser;
     configJSON['ml-admin-pass'] = settings.marklogicAdminPass;
-    configJSON['ml-app-user'] = settings.marklogicAdminUser; //THIS NEEDS TO CHANGE
-    configJSON['ml-app-pass'] = settings.marklogicAdminPass; //THIS NEEDS TO CHANGE
+    configJSON['ml-app-user'] = settings.appName + '-user';
```

Comment:
Safe for slush generator, as the line is commented out in build.properties by default, but worth considering for the (new) gulp init-local/dev/prod..